HttpFs.IntegrationTests/Tests.fs

```diff
@@ -332,6 +332,14 @@ let tests =
 
       Expect.equal actual expected "bytes should be equal"
 
+    testCase "when there is no content-type, charset should be none" <| fun _ ->
+      use response =
+        Request.create Get (uriFor "/NoContentType")
+        |> getResponse
+        |> run
```

Comment:
Can you write this as `testCaseAsync` to avoid deadlocking the test-suite, please?

Reply:
I can of course. I was just doing what all the other tests did, didn't want to break the already set convention.

Reply:
You're right, I need to go through the code-base and correct it. These tests were written before we added async support to expecto.